docs/user-guide/deploying-application/image-labels-and-comments.md

```diff
@@ -0,0 +1,110 @@
+# Applying Labels and Comments
+
+## Introduction
+
+Typically in a CI pipeline, you build [container images](../../reference/glossary.md#image), and the number of images gradually increases over a period of time.
+
+Assume `azurecr.io/web-server` is your container [image repository](../../reference/glossary.md#repo) and it has multiple images. Now, differentiating one `web-server` image from another or remembering a specific `web-server` image from the repository might become a tedious task. 
```

Comment:
I can seperate repositories based on the microservices, seperate repo for `web-server` for what usecases I still need this feature?

Reply:
We have now removed the example of repo and mentioned few use cases upfront.